tueplots/bundles.py

```diff
@@ -236,6 +236,20 @@ def iclr2024(*, usetex=True, rel_width=1.0, nrows=1, ncols=1, family="serif"):
     return {**font_config, **size, **fontsize_config}
 
 
+def probnum2025(*, column="half", nrows=1, ncols=1, family="serif"):
```

Comment:
Do I remember correctly that the figure captions are in sans-serif? If so, what do you think about a "sans-serif" default here? This way, the labels would match the caption, not the main text.

Reply:
I see... I think it's reasonable to assume that most labels don't contain nontrivial maths. If one needs serif fonts, the family can be adjusted. If someone complains, we can consider changing this default in the future :)  

Thanks!